### ISSUE TITLE
OMERO.web image includes several other apps

### ISSUE THREAD

**Issue by manics** [openmicroscopy/omero-test-infra#29]
The `openmicroscopy/omero-web-standalone` Docker image includes several bundled web-apps apps. This could potentially lead to conflicts when testing web applications with this repo: https://github.com/openmicroscopy/omero-test-infra/blob/9c607ce52c9cf153b98bd7e55211cfe2aeb352d5/.env#L11

**Comment by joshmoore** [comment]
Good point. I definitely didn't consider that when you opened your previous PR. Options I can think of:

 * split `standalone` into two images (one with and one without the apps)
 * have a flag to disable/reset the automated configuration
 * duplicate standalone's logic here
 * ...
